src/assets/logos/organizations-small-logo.svg

```diff
@@ -0,0 +1,9 @@
cQNyOAAKZFjjr35h0MK7C2ECw20b8xynN3riBuB0BBDIsYO32uHcphupjA6G8iZ9Z/yMfCkfKl/kfAgjoErD2ZbN9feyjg7BRiQLB3mGnzdDC2/xThxd1SVAtAkUXsE+blUM3J/1OhNgPN9Vz+ncuRuaRo1v9l6OETz+uqL+NywggkCmB8M1Je+yODQfSVJUqEGoDu5+OLzOT5g7/9y3GuuuMs1fxjsaaDr8R6IeAfd/f6xvl/6LoJfOJZb/zLcbJtJV0FAhp72Q+y//r1eNuPut3e91rr7u6L4a4NJ9ZXJq7JL020WsISQdjOQQQ0C1AIOieP6pHQFSAQBDlZDAEdAsQCLrnj+oREBUgEEQ5GQwB3QIEgu75o3oERAUIBFFOBkNAtwCBoHv+qB4BUQECQZSTwRDQLUAg6J4/qkdAVIBAEOVkMAR0CxAIuueP6hEQFSAQRDkZDAHdAgSC7vmjegREBQgEUU4GQ0C3AIGge/6oHgFRAQJBlJPBENAtQCDonj+qR0BUgEAQ5WQwBHQLEAi654/qERAVIBBEORkMAd0CBILu+aN6BEQFCARRTgZDQLcAgaB7/qgeAVEBAkGUk8EQ0C1AIOieP6pHQFSAQBDlZDAEdAsQCLrnj+oREBUgEEQ5GQwB3QIEgu75o3oERAUIBFFOBkNAtwCBoHv+qB4BUQECQZSTwRDQLUAg6J4/qkdAVIBAEOVkMAR0CxAIuueP6hEQFSAQRDkZDAHdAgSC7vmjegREBQgEUU4GQ0C3AIGge/6oHgFRAQJBlJPBENAtQCDonj+qR0BUgEAQ5WQwBHQLEAi654/qERAVIBBEORkMAd0CBILu+aN6BEQFCARRTgZDQLcAgaB7/qgeAVEBAkGUk8EQ0C1AIOieP6pHQFSAQBDlZDAEdAsQCLrnj+oREBUgEEQ5GQwB3QIEgu75o3oERAUIBFFOBkNAtwCBoHv+qB4BUQECQZSTwRDQLUAg6J4/qkdAVIBAEOVkMAR0CxAIuueP6hEQFSAQRDkZDAHdAgSC7vmjegREBQgEUU4GQ0C3AIGge/6oHgFRAQJBlJPBENAtQCDonj+qR0BUgEAQ5WQwBHQLEAi654/qERAVIBBEORkMAd0CBILu+aN6BEQFCARRTgZDQLcAgaB7/qgeAVEBAkGUk8EQ0C1AIOieP6pHQFSAQBDlZDAEdAsQCLrnj+oREBUgEEQ5GQwB3QIEgu75o3oERAUIBFFOBkNAtwCBoHv+qB4BUQECQZSTwRDQLUAg6J4/qkdAVIBAEOVkMAR0CxAIuueP6hEQFSAQRDkZDAHdAgSC7vmjegREBQgEUU4GQ0C3AIGge/6oHgFRAQJBlJPBENAtQCDonj+qR0BUgEAQ5WQwBHQLEAi654/qERAVIBBEORkMAd0CBILu+aN6BEQFCARRTgZDQLcAgaB7/qgeAVEBAkGUk8EQ0C1AIOieP6pHQFSAQBDlZDAEdAsQCLrnj+oREBUgEEQ5GQwB3QIEgu75o3oERAUIBFFOBkNAtwCBoHv+qB4BUQECQZSTwRDQLUAg6J4/qkdAVIBAEOVkMAR0CxAIuueP6hEQFSAQRDkZDAHdArbT8u/f+N6aKeu+4pz9vB9j2Pn/rHHDzpklnY7JegggkE7AWvO+M3bCH8gTfs0Ja93fB5x94RfPL38z3UiVpVMFwubRc1caN3WfX3XUH/jXd3KHrIMAAt0X8EHxT38vY8YOHHhybOk7Se8xUSBs2eSWlCbPfs+Y0sM8AkhKy3II9F8gPIIwJtobLbziZwcPWX+5/U9sIGwZPX3rdMkeNMataj8UtyKAQHYF7MkFkdtycGzouXY1tnxR0fkXBzZvPLPTh8FhwqAdIbchoEHArQrHcjimw7HdquKmNzy0yV32weSZp/yKm1qtyPUIIKBTwFp76LKFy76575D9oHEL5jxCCOlBGDQy8TcC+REI/9BXj/E5DwjmBMI9o2d/yCOD/Ew+W4JAM4FwjIdjvfG2WQlRfQExvGYw6/rGlfgbAQTyIGCdf6Hx9voXGmcO/NBanL505j+8gJiHiWYbEEgq4LsPg8vW1lqSM08ZKu8zoLWYlJHlEMiHgFtVOfYrW1N+hFB9B+KbvOkoH1PMViCQRqD85iU7sCa8o7H8CMG5qfsJgzSELItAfgTKx37lIwmmHAj+YcLG/GweW4IAAh0IjIZ1bPjU4iXn/IuJ/CCAQJEFBq1dG4WPMBcZgW1HAIGKQMiCqPp9BpgggEDBBUIWhNcQhgvuwOYjgEBFYDgK33SEBgIIIBCyIApfewYFAgggELLAv4ZgfDDwgwACRRcIWRAZa98uOgTbjwACXsBngX/KUP62VjwQQKDgAiELQpchfH0zPwgggMBEFL7HHQcEEEAgZEEUTuoABQIIIBCyIApneKme1AERBBAoqEDIgJAF5U87eoOxgjqw2QggUBEoZ0AlEPzpnipneMEGAQSKJlD9gpQDYbvLgVA591u0t2gQbC8CCASBaG/t/I+1pwwmnPvNvzPhJEAIIFAkAXuycuxXtnkmEMK3roZzv/lQ4K3MRdof2NYCC5S/hn1L7RuXA8RMIIQ/wvez+9M87QqX+UEAgXwLhGO9/pwMYWv9uxVn/4RTud0zeub3nL1ptgt/IZAngXB+xyfGln3V/571jGDWI4SwwWGBcCLIsEKeANgWBBCoCIRju3qMzwqDcOucRwg1tMojhfJ5HndyareaCr8R0CxgnQ+DXU+MXfHj8A9/sy1pGQi1havnezzIKd5qIvxGQKOAP2Wbbxo0vmbQuCWxgRBWCOd9rJzuqfQwJ3RpJORvBLIrUHnDYbQ3tBbruwmtKk4UCLWVwynfwlme/EobfTBcX7ue3wggkC2B8NkEX9GY8e9Crr3pKEmFqQKhfsBwgpfwPe7Vr3Ef9k9IhsN3svEIol6Jywh0VyA8AnDGTvgDOXyvyUT4CHP41GL4oFIn9/x/eS/yVXFBDfYAAAAASUVORK5CYII="/>
```

Comment:
It looks like you're embedding an encoded image into an svg. Will this work as a regular svg or no? If not, why don't you just use a png instead?